force-app/main/default/classes/RecurrenceFactory.cls

```diff
@@ -0,0 +1,17 @@
+public with sharing class RecurrenceFactory implements RecurrenceFactoryInterface {
+    @TestVisible
+    private RecurrenceRule recurrenceRule = new RecurrenceRule();
+
+    public RecurrenceIteratorInterface makeIterator(String rRule) {
```

Comment:
makeIterator sounds strange to me.  getIterator?  createIterator?  Curious if anyone agrees.

Reply:
I went with make because of the factory pattern but cool with whatever.

Reply:
Sounds good - we can stick with makeIterator